README.md

```diff
@@ -125,7 +152,7 @@ Installing on macOS:
   ```
   brew install doxygen graphviz
   ```
-  For Latex we recommend MacTeX (http://www.tug.org/mactex/).
```

Comment:
These are required only if you want to build documentation. And I actually disabled PDF generation (it was a fairly useless doc) and thus LaTeX is no longer required. And the same for Ubuntu.